### ISSUE TITLE
vr side panel transitions

### ISSUE THREAD

**Issue by sirkitree** [davexoxide/rendervideositemap#8]
Having a decent amount of text, even faded, I find undesirable. What do you think about during the fade transition they move out a bit on the x axis?

**Comment by davexoxide** [comment]
Yeah if we can get each sidebar as a self-contained entity and it all moves together then I agree fade and slide away is nice eye candy.
